packages/runtime/src/slate/BlockPlugin/index.ts

```diff
@@ -32,3 +36,68 @@ export function withBlock(editor: Editor) {
 
   return editor
 }
+
+export const BlockPlugin = createRichTextV2Plugin({
+  withPlugin: withBlock,
+  controls: [
+    createRichTextV2PluginControl({
```

Comment:
It seems weird that there's a special function to create a control specifically for the rich text v2 plugin. It should be able to take control definitions directly, without having to wrap them with anything.

Also, as I mentioned in another comment, I think we should just take a single control and use `Shape` if you want multiple